### DISCUSSION TITLE
[SOLVED] [GUI] NVMe SSD is unaccessible midway plotting (reproduced in 2 rigs, 3 NVMe SSDs)

### DISCUSSION THREAD

**Discussion by grinbucks** [Chia-Network/chia-blockchain#4459] [Support]
GUI stopped plotting with this error "Only read 0 of 262134 bytes at offset 0 from "T:\\plot..."

I found out that the SSD was no longer accessible  with "A device which does not exist was specified" error.

On the Disk Management, there appeared to be 1mb unallocated space on that disk (see screenshot). 
Before plotting, there was only 1 partition and there was no unallocated space at all.

![Screen Shot 2021-05-03 at 1 06 42 PM](https://user-images.githubusercontent.com/83256537/117648237-2e129280-b1b8-11eb-9d92-9d6f952aa5e5.png)

The disk appears back to normal (1 partition, no unallocated space) after restart.

But the problem persists when plotting.

The error is reproduced in both single and parallel plottings (2,3,4 plots parallel)

The error is reproduced in 2 rigs (B550/Ryzen 3900X and B250/i7-6700)

The error is reproduced in 3 nvme SSDs: 2x Transcend S220 2TB (Gen3) and 1x Aorus 2TB (Gen4)

Before this error, all three SSDs were doing fine plotting for 1 week or so.

GUI versions 1.1.3, 1.1.4, 1.1.5

- OS: Win 10 
- OS Version/Flavor: 20H2 Build 19042.631
- CPU: i7-6700 (with B250 mobo, 40GB DDR4) and Ryzen 3900x (with B550 mobo, 72GB DDR4)

Any help is highly appreciated.

**Comment by michael-trinity** [comment]
@grinbucks try using crystaldiskinfo and see the health of it while plotting, it might just get too hot.

**Comment by grinbucks** [comment]
Noted with thanks @michael-trinity Just restarted plotting and monitoring the temp on rig B550/3900x. Will update on the result.

> @grinbucks try using crystaldiskinfo and see the health of it while plotting, it might just get too hot.

**Reply by grinbucks** [reply]
I have sucessfully managed the NVMe temp between 36-45 and the error did not occur in the B550/3900x rig. I have not done so in the older rig. But I think it's safe to assume that temp is the problem. Thanks again @michael-trinity. Your quick response is highly appreciated.

**Reply by michael-trinity** [reply]
glad it got solved :) temperatures can get very hot without heatsinks

**Reply by webs7882** [reply]
I'm having this same problem, worked fine for two weeks now failing regularly with same errors. Would be interested in how you managed the temperature. I have installed a heat sink and even tried putting the NVME into an external case with an ice pack 🥶

**Comment by dreamboostme** [comment]
I have a similar problem.
XPG SX8200 Pro ASX8200PNP-2TT-C 2TB Disk

I worked for 1-2 weeks, then I started to fall off in Windows.
In Linux, if there is a read error, there is still a chance that the raft will complete successfully. But sooner or later the disk will also fall off, and you can no longer mount it without rebooting.

I was looking for a way to implement hot-plugging to remount it without rebooting, but it didn't work.

It may have overheated earlier. Since now the temperature after installing the cooler does not rise above 40 Celsius during the next failure.

I wanted to reflash, but the latest firmware is already there.

At the moment, I have marked out part 280Gib for 1 plot and am observing, in the hope that this part will not get the "pages" that cause the crash.

**Comment by cyperbg** [comment]
I have the same problem with Firecuda 510 2TB and Aorus 1TB.
Both are plotting for 24 hours and then suddenly read/write errors and “A device which does not exist was specified” error
If I try to delete volume or format in Disk Management they disappear.
After a restart they work fine for like a day.
Both have heatsinks + 140mm fan blowing at them + aircon in the room = temperatures are always below 50.
Any ideas?

**Comment by BrandtH22** [comment]
@grinbucks , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
